diff --git a/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.esm.js b/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.esm.js
index 548c267..5f8e356 100644
--- a/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.esm.js
+++ b/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.esm.js
@@ -4619,7 +4619,10 @@
   useDev(useDevSetupWarning);
 
   var _useState = useState(false),
-      isEnabled = _useState[0],
-      setIsEnabled = _useState[1];
-
-  useEffect(function () {
+    isEnabled = _useState[0],
+    setIsEnabled = _useState[1];
+  useEffect(function () {
+    if (isEnabled) return;
     var unsub = dragDropManager.subscribe(function () {
       return setIsEnabled(true);
     });
@@ -4627,7 +4630,7 @@
     return function unsubscribe() {
       unsub();
     };
-  }, []);
+  }, [isEnabled]);
 
   if (!isEnabled) {
     return null;

